src/components/LocationAutocomplete/LocationAutocomplete.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { LoadScriptNext } from '@react-google-maps/api';
+import { LoadingSpinner } from '../../index';
+import LocationAutocompleteRenderer from './LocationAutocompleteRenderer';
+
+const LOAD_SCRIPT_LIBRARIES = ['places'];
+
+const LocationAutocomplete = props => {
+    const { apiKey, language, region, additionalGoogleProps, ...rest } = props;
+
+    return (
+        <LoadScriptNext
```

Comment:
We would just reimplement this LoadScriptNext. I don't really see the point. What return values would you need?

Reply:
We can use hooks without returned values, like useEffect. For me, it is more consistent, if we use hooks so why we didn't use it here. Other points, when I use a component in most cases I expect it will return some UI, now it is just the extra wrapper. The advantage of hooks is also that we get rid of nested hell :) when one component inside another component and so on.. But I don't sink that will blocker, it is just a proposal.